ape_vyper/compiler.py

```diff
@@ -88,22 +88,32 @@ def compile(self, contract_filepaths: List[Path]) -> List[ContractType]:
                 if pragma_spec is not pragma_spec.select(self.installed_versions):
                     vyper_version = pragma_spec.select(self.available_versions)
                     if vyper_version:
-                        vvm.install_vyper(vyper_version, show_progress=True)
+                        try:
+                            vvm.install_vyper(vyper_version, show_progress=True)
+                        except Exception as e:
+                            raise Abort(f"Unable to install Vyper version: {vyper_version}") from e
                     else:
                         raise Exception("No available version to install")
             else:
                 if not self.installed_versions:
-                    vvm.install_vyper(max(self.available_versions), show_progress=True)
+                    try:
+                        vvm.install_vyper(max(self.available_versions), show_progress=True)
+                    except Exception as e:
+                        raise Abort(
+                            f"Unable to install Vyper version: {max(self.available_versions)}"
+                        ) from e
                 vyper_version = max(self.installed_versions)
```

Comment:
Since you have nothing installed you already know what the only installed version is gonna be, and this cleans up the lint a bit
```suggestion
                    vyper_version = max(self.available_versions)
                    try:
                        vvm.install_vyper(vyper_version, show_progress=True)
                    except Exception as e:
                        raise Abort(f"Unable to install Vyper version: {vyper_version}") from e
```